app/src/main/java/org/projectbuendia/client/ui/chart/ChartRenderer.java

```diff
@@ -158,11 +158,9 @@ public void render(Chart chart, Map<String, Obs> latestObservations,
         void addObservations(List<Obs> observations) {
             for (Obs obs : observations) {
                 if (obs == null) continue;
-                ObsPoint point = obs.getObsPoint();
-                if (point == null) continue;
```

Comment:
Ping, 
i didn't understand this change quiet well this change. Could you explain me better? 
I`ve noticed that this was point which was blocking the counting but  instead of removing it, i'm was trying to assign an ObsPoint to the executed order. Isn't  that make sense ?